graviti/portex/factory.py

```diff
@@ -493,19 +524,43 @@ def __call__(self, **kwargs: Any) -> Fields:
         )
 
 
+def object_unpack_factory_creator(decl: str, is_ptype: bool = False) -> VariableFactory:
+    """Check whether the input have object unpack grammar and returns the corresponding factory.
+
+    Arguments:
+        decl: A decalaration dict.
+        is_ptype: Whether the object unpack indicates a Portex type.
+
+    Raises:
+        ValueError: When the object unpack grammar is incorrect.
+
+    Returns:
+        A ``VariableFactory`` instance or None according to the input.
+
+    """
+    if not decl.startswith("$params."):
```

Comment:
is that possible to put a real mapping after `"+"` rather than use "$params."?

Reply:
I don't see that necessary.